doc/classes/NavigationPathQueryParameters3D.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="NavigationPathQueryParameters3D" inherits="RefCounted" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Parameters to be sent to a 3D navigation path query.
+	</brief_description>
+	<description>
+		This class contains the start and target position and other parameters to be used with [method NavigationServer3D.query_path].
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="calculate_path_length" type="bool" setter="set_calculate_path_length" getter="get_calculate_path_length" default="false">
+			If [code]true[/code], the length of the path will be calculated as the sum of the distances between the path points.
+		</member>
```

Comment:
Couldn't you just do this on demand? (while changing the name of the getter from `get_path_length()` to `find_path_length()` or `calculate_path_length()` to make it clear there is a calculation involved).

That would make this query parameter redundant.

Reply:
Well the idea was to have any kind of path stat calculation as part of the query so the stats can be calculated in a threaded / async manner by the NavigationServer when the parameters for it is set true (so already on demand). Since the async was removed from this pr this part is now missing and it looks like a leftover that could be replaced with a static function anywhere.

Reply:
I will remove it completely from this pr, maybe add as a static helper function later somewhere.